react/src/pages/Page401.tsx

```diff
@@ -38,7 +38,7 @@ const Page401 = () => {
           <Button
             size="large"
             type="primary"
-            onClick={() => webuiNavigate('/start')}
+            onClick={() => webuiNavigate('/data')}
```

Comment:
If you want to navigate `/data` instead of `/start` when PALI mode, please add redirect  it in browserRouter in App.tsx. ([ref](https://github.com/lablup/backend.ai-webui/blob/main/react/src/App.tsx#L190-L200))